contracts/child/ChildChain.sol

```diff
@@ -42,6 +42,12 @@ contract ChildChain is Ownable, StateSyncerVerifier, StateReceiver {
 
   constructor () public {}
 
+  // can be called by anyone to set owner
+  function initConstructor ()
+    public {
+    tokens[0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0] = 0x0000000000000000000000000000000000001010;
```

Comment:
Why hard coding token address? How we are going to test it in testnets?

Reply:
with the current construction will need to hard code test token address first. unless we come up with something runtime.